### PR TITLE
Remove the svfsi_plus docs and fix some formatting problems.

### PR DESCRIPTION
**Pull request by ktbolt**
I fixed some formatting problems, added a `Beta` version note and removed `vfsi_plus` docs, whatever that was.

### CODE REVIEW

File: documentation/svfsiplus/solver-input-file/linear_solver_parameters/readme.md [hunk 1]
```diff
@@ -54,10 +54,10 @@ The value of <i>linear_solver_type</i> can be
 </div>
 
 <!-- --------------------------------- -->
-<!-- ---------- Description ---------- -->
+<!-- ----- Liner solver parameters --- -->
```
Comment (dcodoni) [inline documentation/svfsiplus/solver-input-file/linear_solver_parameters/readme.md]: typo: Linear

File: documentation/svfsiplus/solver-input-file/linear_solver_parameters/readme.md [hunk 2]
```diff
@@ -54,10 +54,10 @@ The value of <i>linear_solver_type</i> can be
...
 <!-- --------------------------------- -->
 
-<h5> Description </h5>
+<h4 id="liner_solver_parameters"> Liner Solver Parameters </h4>
```
Comment (dcodoni) [inline documentation/svfsiplus/solver-input-file/linear_solver_parameters/readme.md]: typo: linear

File: documentation/svfsiplus/user-guide/fluid_solid_interaction/introduction/readme.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
\ No newline at end of file
+**svFSIplus** utilizes Arbitrary Lagrangian-Eularian (ALE) method to perform FSI. As the name implies, this method is a combination of the Eularian and Lagrangian descriptions of motion that is particularly well suited for FSI problems. In the Eularian description, the computational mesh stays fixed and the motion is characterized as velocities flowing past the grid nodes. The Eularian description is typical for most fluid dynamics problems without FSI. While in the Lagrangian description, the mesh nodes move exactly with the motion of the fluid or solid and thus are often characterized as *moving domain* problems. Lagrangian problems are also often described in terms of the *reference* configuration (the initial computational geometry before any motion) and the *current* configuration (the current state of the geometry and mesh nodes). ALE combines these descriptions into a formulation that is convenient for describing FSI problems, where the mesh motion does not exactly match with the fluid motion, but instead a *mesh velocity* term is added to the convective term in the Navier-Stokes equations. More information can be found in the literature.
```
Comment (dcodoni) [inline documentation/svfsiplus/user-guide/fluid_solid_interaction/introduction/readme.md]: typo: Eulerian

File: documentation/svfsiplus/user-guide/fluid_solid_interaction/setting_up_simulation/readme.md [hunk 4]
```diff
@@ -51,4 +51,4 @@ The “Simulation Parameters” panel allows specification of general parameters
 
 Finally, on the “Run Simulation” panel, click “Create Input File” to create the necessary files to run your simulation. If you wish to run with more than one processor, increase the slider accordingly. Finally, click “Run” to run your simulation.
 
-svFSI will create a directory called n-procs, where n is the number of MPI tasks for the simulation. This directory will contain .vtu files that with values of all requested fields, as well as a log file called history.dat and averages of various quantities over time.
+svFSIplus will create a directory called n-procs, where n is the number of MPI tasks for the simulation. This directory will contain .vtu files that with values of all requested fields, as well as a log file called history.dat and averages of various quantities over time.
```
Comment (dcodoni) [inline documentation/svfsiplus/user-guide/fluid_solid_interaction/setting_up_simulation/readme.md]: sentence, maybe remove _that_: This directory will contain .vtu files _that_ with values of all requested fields

File: documentation/svfsiplus/user-guide/hemodynamics/introduction/readme.md [hunk 5]
```diff
@@ -1,3 +1,3 @@
...
+Modeling of cardiovascular flow is the main function of **SimVascular**. Currently, the default flow solver of **SimVascular** is [**svSolver**](flowsolver.html). All of the features in **svSolver** exists in **svFSIplus** including RCR boundary condition, Coupled Momentum Method, GenBC etc. Also, we have carefully tested the new solver to make sure it produce the same results as **svSolver**. In this section, we won't dwell on the similarities between these two solvers, and instead will focus two new features in flow simulation using **svFSIplus**, i.e., non-Newtonian flow model and simulation of prescribed motion.
```
Comment (dcodoni) [inline documentation/svfsiplus/user-guide/hemodynamics/introduction/readme.md:3]: sentence: instead will focus _on_ two new features

**Review by dcodoni — COMMENTED** [review]
Thanks @ktbolt for doing this! I have just very few comments (typos)

File: documentation/svfsiplus/user-guide/hemodynamics/introduction/readme.md (hunk 5 above)
Comment (ktbolt) [reply documentation/svfsiplus/user-guide/hemodynamics/introduction/readme.md:3]: @dcodoni Thanks for actually reading the documentation! I made the suggested changes and learned how to spell `linear`. 

The `User Guide` sections were taken from the svFSI documentation just to have something there. I've not edited them, need to be rewritten from scratch I think.

**Review by dcodoni — APPROVED** [review]
Good!

**Pull request merged**